editor/sidebar/post-visibility/index.js

```diff
@@ -111,12 +129,17 @@ class PostVisibility extends Component {
 					aria-expanded={ this.state.opened }
 					className="editor-post-visibility__toggle button-link"
 					onClick={ this.toggleDialog }
+					ref={ toggle => this.toggle = toggle }
 				>
 					{ getVisibilityLabel( visibility ) }
 				</button>
 
 				{ this.state.opened &&
-					<div className="editor-post-visibility__dialog">
+					<div className="editor-post-visibility__dialog"
+						onKeyDown={ this.handleKeyDown }
```

Comment:
Warning:

```
138:6   warning  Visible, non-interactive elements should not have mouse or keyboard event listeners
```

Might explain why this was avoided for DropdownMenu . Is it fair to disable the rule here?